docs/source/changes.md

```diff
@@ -125,8 +130,8 @@ authors:
 - `epochs_tmin` and `epochs_tmax` were named incorrectly in some test config
   files.
   ({{ gh(340) }} by {{ authors.crsegerie }})
-- We now reject bad epochs by using [`ica_reject`][config.ica_reject] before 
-  producing the "overlay" plots that show the evoked data before and after 
+- We now reject bad epochs by using [`ica_reject`][config.ica_reject] before
```

Comment:
remove

Reply:
No this one is fine, this is unrelated to autoreject and was the plotting fix that @crsegerie added last week